go.rst

```diff
@@ -321,7 +299,32 @@ Wahl stehen.
      die restlichen Plätze nach Anzahl der Stimmen in der ersten Runde
      besetzt. Bei Gleichstand entscheidet das Los.
 
-9. Abwahlen sind auch bei Abwesenheit der betroffenen Person möglich und
+7.2  Der so bestimmten Gruppe muss anschließend das Vertrauen ausgesprochen werden. Dies geschieht 	falls nicht anders gewünscht per Akklamation.
+     Sind die ersten sechs Personen genannter Gruppe vom gleichen Geschlecht,
+     ersetzt die Person eines anderen Geschlechts mit den meisten Stimmen die
+     sechste Person in der Rangfolge.
+     Sollten sich nur Personen eines Geschlechts beworben haben, ist diese
+     Regelung irrelevant.
+
+     Bei weniger als sieben sich bewerbenden Speziemen der Gattung homo sapiens sapiens muss der
+     kompletten Gruppe das Vertrauen mit Zweidrittelmehrheit ausgesprochen werden, damit sie als
+     gewählt gilt.
+     Die Wahl durch Zustimmung entfällt hierbei.
+
+7.3  Eine Personaldebatte findet nicht statt, die Kandidierenden dürfen sich jedoch dem Plenum
```

Comment:
Die Aufteilung in nummerierte Absätze erscheint mir hier ziemlich arbiträr. Was ist der Gedanke dahinter, die Personaldebatte und die Bekanntgabe der Abstimmungsergebnisse in den gleichen Absatz zu packen?

Reply:
kann ich dir ehrlich gesagt so jetzt nicht sagen. Haben wir irgendwann spät nachts so gemacht. Können wir gerne im AK darüber diskutieren.